src/parser.ml

```diff
@@ -237,97 +194,99 @@ type t =
   | Lthematic_break
   | Latx_heading of int * string * attributes
   | Lsetext_heading of int * int
+      (** the level of the heading and how long the underline marker is *)
```

Comment:
Should we consider inline records here if they don't make later code more verbose?

Reply:
Good idea!